state-chain/pallets/cf-swapping/src/tests.rs

```diff
@@ -1802,6 +1817,7 @@ fn swap_with_custom_broker_fee(
 	to: Asset,
 	amount: AssetAmount,
 	broker_fee: BasisPoints,
+	ingress_fee: AssetAmount,
```

Comment:
This is 0 in every test. We should add some tests where it's not 0.

Reply:
and as stated before - one of these tests should be a CcmTransfer scheduled from channel to ensure the ingress fee is set there